create-release-notes/index.js

```diff
@@ -112,6 +125,31 @@ function convertToHtml(fieldData) {
     return "";
 }
 
+async function createReleaseNote(webflowToken, product, releaseName, releaseNotesContent) {
+    const webflowAuthToken = `Bearer ${webflowToken}`.toString("base64");
+    const webflowHost = "api.webflow.com";
+    const currentDate = new Date();
+    const body = {
+        cmsLocaleId: "653ad721e882f528b341eaa5",
+        isArchived: false,
+        isDraft: false,
+        fieldData: {
+            "release-date": currentDate,
+            "removed-flag": false,
+            "product": product,
+            "content": releaseNotesContent,
+            "name": releaseName
+        }
+    }
+    const createResponse = JSON.parse(await post(webflowHost, "/v2/sites/62e28cf08913e81176ba2c39/collections/66c653f35fee4b88416da2b2/items", webflowAuthToken, body));
```

Comment:
Should we make this a github action var?

Reply:
not sure if it's worth doing that. Since currently there is just one site and we only are working with one collection. If we do something else in the future we can look at either creating variables for multiple sites and/or collections. However, it's hard to anticipate if and what will be needed in the future.

Reply:
I was thinking more along the lines of the site gets accidentally deleted and it has a new id when they bring it back

Reply:
I guess since its in the toolbox, that isn't so bad either